eth-contracts/.prettierrc.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+    tabWidth: 2,
+    semi: false,
```

Comment:
why set `semi` to false? 
I checked [airbnb](https://github.com/airbnb/javascript#semicolons) and [google](https://google.github.io/styleguide/jsguide.html#formatting-semicolons-are-required) style guide, they all recommend write semicolons in the code instead of relying on `Automatic Semicolon Insertion`.

Reply:
by borrowing from other similar projects,  I use the similar config file. I am not sure this is better config